File: mobile-client/ios/Contract/Catalog/Browse/CatalogNodeSummary.swift
import Foundation

// Marketing America Corp. Oleksandr Tishchenko
public struct CatalogNodeSummary: Sendable, Equatable {
    public let nodeId: String
    public let parentNodeId: String?
    public let title: String
    public let slug: String?
    public let childCount: Int
    public let productCount: Int?
}

File: mobile-client/android/Contract/Catalog/Browse/CatalogNodeSummary.kt
package app.mobiling.client.contract.catalog.browse

/**
 * Marketing America Corp. Oleksandr Tishchenko
 */
data class CatalogNodeSummary(
    val nodeId: String,
    val parentNodeId: String?,
    val title: String,
    val slug: String?,
    val childCount: Int,
    val productCount: Int?,
)
